### DISCUSSION TITLE
(How) Can I install Bolt 5 via FTP?

### DISCUSSION THREAD

**Discussion by adastrame** [bolt/core#3381] [Q&A]
This might be a silly question from someone who's been out of the web dev business for 10 years - how can I install Bolt 5 via FTP? I do not know how to use composer or deploy from local environments, and I don't particularly need any of that for the tiny projects I have in mind. 

I used Bolt 3 via manual FTP installations in the past and would just like to do that again with the latest version.

**Comment by felixfruehstueck** [comment]
Hi adastrame,
I asked that exact same question in the Slack channel. Looks like it's not possible.
Here's how I solved the situation:

- Set up a blank Bolt 5 project with composer, using SQlite DB.
- Copied over the Bolt 3 contenttypes definitions and Bolt 3 theme, then fixed all the errors that popped up
- Re-created all contents from Bolt 3 (server) in my local Bolt 5 copy
- Deleted the whole server (inkl. Bolt 3 mysql DB)
- Uploaded everything from my local Bolt 5 via FTP.

I know it's not pretty but I could not find another way (it's not my webspace).
Especially, it does not seem to be possible to convert the old Bold 3 DB to new Bolt 5 schema directly on the server. Looks like it's not even possible to set up a blank mysql db on the server without SSH access.

Hope that helps!
Best,
Felix
